go-selinux/selinux_linux.go

```diff
@@ -1083,6 +1084,50 @@ func chcon(fpath string, label string, recurse bool) error {
 		return nil
 	}
 
+	exclude_paths := map[string]bool{
+		"/":           true,
+		"/bin":        true,
+		"/boot":       true,
+		"/dev":        true,
+		"/etc":        true,
+		"/etc/passwd": true,
+		"/etc/pki":    true,
+		"/etc/shadow": true,
+		"/home":       true,
+		"/lib":        true,
+		"/lib64":      true,
+		"/media":      true,
+		"/opt":        true,
+		"/proc":       true,
+		"/root":       true,
+		"/run":        true,
+		"/sbin":       true,
+		"/srv":        true,
+		"/sys":        true,
+		"/tmp":        true,
+		"/usr":        true,
+		"/var":        true,
+		"/var/lib":    true,
+		"/var/log":    true,
+	}
+
+	if home := os.Getenv("HOME"); home != "" {
+		exclude_paths[home] = true
+	}
+
+	if sudoUser := os.Getenv("SUDO_USER"); sudoUser != "" {
+		if usr, err := user.Lookup(sudoUser); err == nil {
+			exclude_paths[usr.HomeDir] = true
+		}
+	}
```

Comment:
Will this part bring back the issue you ran into that you were trying to solve in https://github.com/opencontainers/selinux/pull/180?

> But some users put homedirectories under /usr, and I see no reason to block them from relabeling.

Reply:
No the #180 was blocking /usr/*
Now we are just blocking /usr or $HOME not $HOME/*

Reply:
Ah, gotcha!